puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/ansible.groovy

```diff
@@ -45,7 +45,8 @@ def runPlaybook(args) {
 
 def writeExtraVars(args) {
     def timestamp = System.currentTimeMillis()
-    def extra_vars_file = 'extra_vars-' + timestamp.toString() + '.yaml'
+    def uuid = UUID.randomUUID()
+    def extra_vars_file = 'extra_vars-' + timestamp.toString() + '-' + uuid.toString() + '.yaml'
```

Comment:
Can you use `File.createTempFile` similar to `System.currentTimeMillis` here?

Reply:
`Scripts not permitted to use staticMethod java.io.File createTempFile` (aka: no)